variables-and-memory/problem-set-variables-and-memory.md

```diff
@@ -244,10 +244,7 @@ all_stores = [
 ]
 
 book_stores = all_stores
-
-for store in book_stores:
-    if store["type"] != "book":
-        book_stores.remove(store)
+book_stores.pop()
```

Comment:
This change looks fine, but out of curiosity, why do the challenges change from multiple-choice to checkbox on this one? multiple-choice does a single response radio button, while checkbox does multiple response checkboxes. I don't see why we would want to change to the multi-response for the last two questions.

Reply:
I think that Learn has a hard time with the single response when the answers are a code block. At least that was my experience in the past.-- but it's not longer the case, so I changed it!